ui/src/timeMachine/reducers/index.test.ts

```diff
@@ -77,13 +78,23 @@ describe('timeMachinesReducer', () => {
         name: '',
         text: 'foo',
         editMode: QueryEditMode.Advanced,
-        builderConfig: {buckets: [], tags: [], functions: []},
+        builderConfig: {
+          buckets: [],
+          tags: [],
+          functions: [],
+          aggregateWindow: {period: 'auto'},
```

Comment:
😬 sorry these reducer tests should have died a long time ago, this is an annoying change to make